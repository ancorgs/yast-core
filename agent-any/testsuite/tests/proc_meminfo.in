        total:    used:    free:  shared: buffers:  cached:
Mem:  131194880 101560320 29634560 35786752 14327808 51138560
Swap: 131600384  3346432 128253952
MemTotal:    128120 kB
MemFree:      28940 kB
MemShared:    34948 kB
Buffers:      13992 kB
Cached:       49940 kB
SwapTotal:   128516 kB
SwapFree:    125248 kB
