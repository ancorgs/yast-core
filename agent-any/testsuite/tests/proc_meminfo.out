($["buffers":13992, "cached":49940, "mem":$["buffers":14327808, "cached":51138560, "free":29634560, "shared":35786752, "total":131194880, "used":101560320], "memfree":28940, "memshared":34948, "memtotal":128120, "swap":$["free":128253952, "total":131600384, "used":3346432], "swapfree":125248, "swaptotal":128516])
